api/v1beta1/vmagent_types.go

```diff
@@ -109,10 +109,6 @@ type VMAgentSpec struct {
 	// cannot be used with HostNetwork.
 	// +optional
 	HostAliases []v1.HostAlias `json:"host_aliases,omitempty"`
-	// PodSecurityPolicyName - defines name for podSecurityPolicy
-	// in case of empty value, prefixedName will be used.
-	// +optional
-	PodSecurityPolicyName string `json:"podSecurityPolicyName,omitempty"`
```

Comment:
Is it safe to delete objects definition from CRD? 

maybe we should keep it as is and remove deprecated fields at `v1` version of API?

Reply:
currently it could be no-op field